curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md

```diff
@@ -80,6 +80,27 @@ assert.deepEqual(tokenize(testStr2, '&', '@'), res2);
 const testStr1 = 'one^|uno||three^^^^|four^^^|^cuatro|';
 const res1 = ['one|uno', '', 'three^^', 'four^|cuatro', ''];
 
+const testStr2 = 'a@&bcd&ef&&@@hi';
```

Comment:
All of these new test cases should be reflected in the `hints` section above. Otherwise they aren't actually tests.